en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -57,11 +57,15 @@ During deployment, Choreo performs a Trivy scan to detect critical vulnerabiliti
 
 ## Environment variables
 
-You can use environment variables in WSO2 Micro Integrator to improve the configuration management, security, portability, and manageability of the Micro Integrator installation. By using environment variables, organizations can simplify the management and maintenance of their micro installations and ensure that they can be quickly and easily updated when required.
+It is recommended to use environment variables when developing integration artifacts with WSO2 Integration Studio to improve configuration management, security, portability, and manageability. By using environment variables, organizations can simplify the management and maintenance of their integrations and ensure that they can be quickly and easily updated when required.
 
-The Choreo DevOps Portal allows you to define configurations and secrets to a deployed integration (as well as other component types). For more information, see [Configurations and secrets](../../devops/devops-portal.md#configurations-and-secrets).
+The component deploy page offers a user-friendly interface for managing environment variables across different environments. For more advanced use cases, the Choreo DevOps portal is available.
```

Comment:
Would you be able to explain what message we need to convey here?

Reply:
The users can manage environment variables for the general use cases through the component deployment page. If they have more advanced use cases, they can do the configurations in Choreo DevOps portal.

Reply:
The component's **Deploy** page allows you to easily manage environment variables across environments via a user-friendly interface. For more information on how you can manage environment variables for more advanced use cases, see [Configurations and secrets](../../devops/devops-portal.md#configurations-and-secrets).

Reply:
Fixed in a1a9d6250d9d356ba65d8826e0ee55a40d3a258e